doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -335,8 +335,9 @@ for the following aspects:
   directly. I.e., updates must be done deleting and re-creating the registration. Please
   see [this issue](https://github.com/telefonicaid/fiware-orion/issues/3007) about this.
 * `idPattern` is supported but only for the exact regular expression `.*`
-  And, right now (2019-03-21), only forwarding of queries is working for registrations with idPatterns.
-  forwarded updates do not work for registrations with idPattern (the plan is to fix this asap). 
+  And, right now (2019-04-24), forwarding for registrations with idPatterns works for queries using `GET /v2/entities`
+  as well as updates using `PATCH /v2/entities/{Entity-ID}/attrs`.
+  More requests will be added to this list.
```

Comment:
I think is better the old wording, without providing the detail of the particular operations. At least until the translation of cases/0787 test so we can have more insight in which operations are supported and which operations doesn't.

Reply:
Until we have a clear path forward, let's wait with these kind of changes. No need to spend time before we are sure what we want exactly

Reply:
... and if there is no need of spending time, why this file was modified? ;)

I'd suggest to go back for its previous version.

Reply:
I was following a perhaps unclear spec and this was my interpretation, a logical one by the way.
I did not see this as spending time for nothing.